pyperf/tests/test_perf_cli.py

```diff
@@ -154,6 +154,26 @@ def test_compare_to_table(self):
         self.assertEqual(stdout.rstrip(),
                          expected)
 
+
+    def test_compare_to_md_table(self):
+        ref_result = self.create_bench((1.0,),
+                                       metadata={'name': 'telco'})
+
+        changed_result = self.create_bench((2.0,),
+                                           metadata={'name': 'telco'})
+
+        stdout = self.compare('compare_to', ref_result, changed_result, '--table',
+                              '--table-format', 'md')
+
+        expected = textwrap.dedent('''
+            | Benchmark | ref      | changed                |
+            |-----------|----------|------------------------|
```

Comment:
Would it be possible to align the first column to the left, and align other columns at the end?

`| --------- | :------: | :--------------------: |`

Reply:
@vstinner Done :)